config/tts.yml

```diff
@@ -0,0 +1,26 @@
+default: &default
+  credentials_file: config/wort-schule-371317-a7ac0879d2d8.json
```

Comment:
What about using a fixed filename here, e.g. `config/google-tts-credentials.json`? And then we can add that specific one to `.gitignore`, so that nobody accidentally commits their credentials.